pkg/controller/nodemaintenance/nodemaintenance_controller.go

```diff
@@ -229,7 +289,7 @@ func (r *ReconcileNodeMaintenance) Reconcile(request reconcile.Request) (reconci
 	return reconcile.Result{}, nil
 }
 
-func (r *ReconcileNodeMaintenance) stopNodeMaintenance(nodeName string) error {
+func (r *ReconcileNodeMaintenance) stopNodeMaintenance(nodeName string, reqLogger *log.Entry) error {
```

Comment:
It would be preferable to use the `log` API directly than to pass `reqLogger` into every function we want output from.  We don't really benefit from having the request name and namespace logged every time.

Reply:
done